doc/release-notes.md

```diff
@@ -36,82 +32,213 @@ from a few minutes to thirty minutes to finish. After the migration, a
+
+### `MNHFTx`
+
+`MNHFTx` will support a new `version` value:
+
+| Version | Version Description                            |
+|---------|------------------------------------------------|
+| 1       | Serialisation of `sig` using legacy BLS scheme |
+| 2       | Serialisation of `sig` using basic BLS scheme  |
+
+## Wallet
+
+## Automatic wallet creation removed
+
+Dash Core will no longer automatically create new wallets on startup. It will
+load existing wallets specified by -wallet options on the command line or in
+dash.conf or settings.json files. And by default it will also load a
+top-level unnamed ("") wallet. However, if specified wallets don't exist,
+Dash Core will now just log warnings instead of creating new wallets with
+new keys and addresses like previous releases did.
+
+New wallets can be created through the GUI (which has a more prominent create
+wallet option), through the dash-wallet create command or the createwallet RPC.
+
+## P2P and Network Changes
+
+## Removal of reject network messages from Dash Core (BIP61)
+
+The command line option to enable BIP61 (-enablebip61) has been removed.
+
+Nodes on the network can not generally be trusted to send valid ("reject")
+messages, so this should only ever be used when connected to a trusted node.
+Please use the recommended alternatives if you rely on this deprecated feature:
+
+- Testing or debugging of implementations of the Dash P2P network protocol
+should be done by inspecting the log messages that are produced by a recent
+version of Dash Core. Dash Core logs debug messages
+(-debug=<category>) to a stream (-printtoconsole) or to a file
+(-debuglogfile=<debug.log>).
+
+- Testing the validity of a block can be achieved by specific RPCs:
+    - submitblock
+    - getblocktemplate with 'mode' set to 'proposal' for blocks with
+    - potentially invalid POW
+    - Testing the validity of a transaction can be achieved by specific RPCs:
+      - sendrawtransaction
+      - testmempoolaccept
+
+## CoinJoin update
+
+A minor update in several CoinJoin-related network messages improves support
+for mixing from SPV clients. These changes make it easier for SPV clients to
+participate in the CoinJoin process by using masternode information they can
+readily obtain and verify via [DIP-0004](https://github.com/dashpay/dips/blob/master/dip-0004.md).
+
+## Remote Procedure Call (RPC) Changes
 
 ### The new RPCs are:
-None
+- In order to support BLS public keys encoded in the legacy BLS scheme, `protx register_legacy`, `protx register_fund_legacy`, and `protx register_prepare_legacy` were added.
+- `cleardiscouraged` clears all the already discouraged peers.
+- The following RPCs were added: `protx register_hpmn`, `protx register_fund_hpmn`, `protx register_prepare_hpmn` and `protx update_service_hpmn`.
+  These HPMN RPCs correspond to the standard masternode RPCs but have the following additional mandatory arguments: `platformNodeID`, `platformP2PPort` and `platformHTTPPort`.
+
```

Comment:
`upgradewallet`, `analyzepsbt`

Reply:
analyzepsbt was in v18. Where was upgradewallet introduced? I somehow missed that.

Reply:
https://github.com/dashpay/dash/commit/5f10b3e5dc9744aedca5cb4b222c777389f86155